SWIG/randomnumbers.i

```diff
@@ -191,6 +191,18 @@ class SobolRsg {
     const Sample<std::vector<Real> >& nextSequence() const;
     const Sample<std::vector<Real> >& lastSequence() const;
     Size dimension() const;
+    %extend{
+      void skipTo(Size n){
+        $self->skipTo(n);
+      }
+
+      std::vector<Size> nextInt32Sequence(){
```

Comment:
`vector<unsigned int>` might avoid the problems you were having with `Size`, and it might be a better match with the underlying interface.

Reply:
yes that could be done